Moderation-Policy.md

```diff
@@ -27,10 +27,9 @@ supported by the Admin team of the Slack organization.
 
 ## Terms
 
-* *Collaborator* refers to any individual with configured write (commit)
-  permissions to any Node.js GitHub organization repository *other than the
-  Moderation Repository*. See [GitHub's access permissions documentation][] for
-  more information.
+* *Collaborator* refers to any individual with configured triage permissions to
+  any Node.js GitHub organization repository. See
+  [GitHub's access permissions documentation][] for more information.
```

Comment:
This links to https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github but perhaps it's better to link to https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization where the roles are identified and explained?

Reply:
```suggestion
  [GitHub's repository roles documentation][] for more information.
```